const Exception = () => {
  return (<div><p>404 Not Found</p></div>)
}

export default Exception